charts/seed-monitoring/charts/core/charts/prometheus/templates/config.yaml

```
@@ -577,6 +594,12 @@ data:
         - __meta_kubernetes_service_name
         action: keep
         regex: blackbox-exporter
+      - target_label: __address__
+        replacement: kube-apiserver:443
+      - source_labels: [__meta_kubernetes_service_name]
+        regex: (.+)
+        target_label: __metrics_path__
+        replacement: /api/v1/namespaces/kube-system/services/${1}:9115/proxy/probe
```

Can we use the port name instead of the port number here?
```suggestion
        replacement: /api/v1/namespaces/kube-system/services/${1}:probe/proxy/probe
```

---

okay